packages/mobile/index.js

```diff
@@ -9,12 +9,15 @@ if (__DEV__) {
 import { AppRegistry } from 'react-native'
 import Logger from 'src/utils/Logger'
 // This needs to happen early so any errors (including in the store) get caught
-import { SENTRY_ENABLED } from 'src/sentry/Sentry'
+import Config from 'react-native-config'
+import { stringToBoolean } from '@celo/utils/lib/parsing'
 import App from 'src/app/App'
 import * as Sentry from '@sentry/react-native'
 import 'react-native-gesture-handler'
 import { Text, TextInput } from 'react-native'
 
+const SENTRY_ENABLED = stringToBoolean(Config.SENTRY_ENABLED || 'false')
```

Comment:
i tried to import this from Config but ran into some runtime errors (typeerror: undefined is not an object evaluating _config.TOS_LINK) - i thought this was relatively low impact so went with this rather than spend time on the errir

Reply:
> i tried to import this from Config but ran into some runtime errors (typeerror: undefined is not an object evaluating _config.TOS_LINK) - i thought this was relatively low impact so went with this rather than spend time on the errir

Ah because of the [require cycles](https://github.com/valora-inc/wallet/issues/1385) we have :/
Looks good then, until we can properly address those.